README.md

```diff
@@ -13,23 +13,22 @@
   <img alt="blueprints banner" src="docs/source/_static/blueprints_banner.png">
 </picture>
 
-Welcome to Blueprints, the cornerstone repository for civil engineering professionals and enthusiasts alike. Blueprints is a collaborative,
-open-source initiative designed to serve as a comprehensive library of code, tools, and best practices, aiming to streamline and innovate the
-workflows within the built environment sector.
+Can someone explain to me why eurocode is not publicly available in code and that all of us are coding eurocode with massive maintenance costs 
+and shortages in technical personel? Well, no longer!
```

Comment:
Spelling mistake - personnel

Reply:
Corrected in 0b05c4961945a3e056a325b506274a867b82ff6a